cmd/crc/cmd/status.go

```diff
@@ -195,11 +195,13 @@ func (s *status) prettyPrintTo(writer io.Writer) error {
 		{"CRC VM", s.CrcStatus},
 	}
 
-	if s.Preset == preset.OpenShift {
-		lines = append(lines, line{"OpenShift", openshiftStatus(s)})
-	}
-	if s.Preset == preset.Microshift {
+	switch s.Preset {
+	case preset.Microshift:
 		lines = append(lines, line{"MicroShift", openshiftStatus(s)})
+	case preset.OKD:
+		lines = append(lines, line{"OKD", openshiftStatus(s)})
+	default:
+		lines = append(lines, line{"OpenShift", openshiftStatus(s)})
```

Comment:
`lines = append(lines, line{s.Preset.ForDisplay(), openshiftStatus(s)})` should do the trick.